model/authorize_test.go

```diff
@@ -51,14 +49,13 @@ func TestAuthIsValid(t *testing.T) {
 	require.NotNil(t, ad.IsValid())
 
 	ad.ClientId = NewRandomString(28)
-	if err := ad.IsValid(); err == nil {
-		t.Fatal("Should have failed Client Id")
-	}
+	require.NotNil(t, ad.IsValid(), "Should have failed Client Id")
 
 	ad.ClientId = NewId()
 	require.NotNil(t, ad.IsValid())
 
 	ad.UserId = NewRandomString(28)
+	require.NotNil(t, ad.IsValid(), "Should have failed Client Id")
 	if err := ad.IsValid(); err == nil {
 		t.Fatal("Should have failed User Id")
 	}
```

Comment:
A new require was added for this, so the old call to `t.Fatal` should be removed

Reply:
updated.